web_client/src/components/CustomRangeSlider.vue

```diff
@@ -10,7 +10,7 @@ export default VRangeSlider.extend({
       } else {
         this._middleDragStart = null;
       }
-      VRangeSlider.options.methods.onSliderMouseDown.call(this, e);
+      (VRangeSlider as any).options.methods.onSliderMouseDown.call(this, e);
```

Comment:
I see an error saying `unexpected any`. Can this be cast to an html element or something similar? There are multiple warnings about these raised by GH actions, you can see them in the Files Changed tab.

Reply:
It wouldn't take as an `HTMLElement` but it appears if I import `VRangeSlider` with an explicit path TS is able to grab the typing.